Header header
bool valid
nrf_project/LaneMsg left_lane
nrf_project/LaneMsg right_lane
int32 n_next_lanes
nrf_project/LaneMsg[] next_lanes
int32 n_obstacles
nrf_project/ObstacleMsg[] obstacles
